scripts/datasets/DET.py

```diff
@@ -0,0 +1,203 @@
+"""Prepare DET(Object detection) datasets"""
+import os
+import argparse
+import tarfile
+from os.path import join, isdir
+import glob
+import xml.etree.ElementTree as ET
```

Comment:
Please follow the code here to import xml,

https://github.com/dmlc/gluon-cv/blob/7d9d6030bd44e3a3a937d565c32468944fc0b2e9/gluoncv/data/pascal_voc/detection.py#L8